target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -115,12 +115,14 @@ engenius,ecb1750|\
 enterasys,ws-ap3705i)
 	ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0"
 	;;
+compex,wpj344-16m|\
 compex,wpj531-16m)
+	_board="${boardname%-*}"
```

Comment:
I have made bad experiences with all these nested definitions in ar71xx.
So, I'd very much prefer to just have two separate cases.

Reply:
done - pushing in a moment